common/config/presets/tari_igor_config.toml

```diff
@@ -0,0 +1,535 @@
+########################################################################################################################
+#                                                                                                                      #
+#                                      The Tari Network Configuration File                                             #
+#                                                                                                                      #
+########################################################################################################################
+
+# This file carries all the configuration options for running Tari-related nodes and infrastructure in one single
```

Comment:
Should this not just be merged with the current config, just adding the igor settings?